README.md

```diff
@@ -85,7 +85,7 @@ Clone this repository to your desired folder:
 👤 **Emeka Emmanuel**
 
 - GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+- LinkedIn: [Emeka Ugboaja](https://linkedin.com/in/emeka-ugboaja-167820226)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p> 
```

Comment:
- [ ] Please the `<figure>` tag specifies self-contained content, like illustrations, diagrams, photos, and code listings, it is more semantic to not use them here
![image](https://user-images.githubusercontent.com/74744787/210816949-cf3ab0ea-bfe9-422a-a797-f66f2a4df6e2.png)

Reply:
✔️